packages/mux-player/package.json

```diff
@@ -43,6 +45,7 @@
     "eslint": "^7.11.0",
     "npm-run-all": "^4.1.5",
     "replace": "^1.2.1",
+    "shx": "^0.3.4",
```

Comment:
Theoretically, we can have `shx` at the top level, and it'll be available for usage in all the workspace packages

Reply:
True. Philosophically, I prefer to add the dependencies where they are actually expected. It means version syncing/upgrades require some additional TLC but makes tracking and removing dependencies that are no longer used easier, and makes it clearer how dependencies are made available for where they're used.

Reply:
I'd lean towards using something like https://github.com/JamieMason/syncpack for those parts of housekeeping.